lms/static/scripts/postmessage_json_rpc/server/server.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+export default class Server {
+  constructor(window_) {
```

Comment:
Using a default argument (`window_=window`) would be more idiomatic than `window_ = window_ || window` in ES6. The former avoids overwriting assignments in some cases where the latter would.

Reply:
Great! Done